Sources/Xcodeproj/pbxproj.swift

```diff
@@ -691,7 +693,7 @@ public func xcodeProject(
         // Otherwise, create an aggregate target.
         let package = packagesByProduct[product]!
         let aggregateTarget = project.addTarget(
-            objectID: "\(package.manifestName)::\(product.name)::ProductTarget", // TODO: use identity instead?
+            objectID: "\(package.manifest.displayName)::\(product.name)::ProductTarget", // TODO: use identity instead?
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
See above